ibm_catalog.json

```
@@ -0,0 +1,116 @@
+          "support_ibm",
+          "integration"
+        ],
+        "keywords": [
+          "code engine",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution"
+        ],
+        "short_description": "Supports provisioning and configuring IBM Code Engine for IBM Cloud.",
+        "long_description": "This architecture supports creating IBM Code Engine with project, apps, domain mappings, secrets, config maps, and service bindings.",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-code-engine/blob/main/README.md",
+        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-code-engine/main/images/code-engine.svg",
+        "provider_name": "IBM",
+        "features": [
+          {
+            "title": "Create IBM Code Engine Project",
+            "description": "Creates IBM Code Engine Project to group, manage and provide access for Code Engine entities such as applications, jobs, and builds."
+          },
+          {
+            "title": "Create IBM Code Engine Apps",
+            "description": "Creates applications to serve HTTP requests or WebSockets as their communications protocol. The number of running instances of an app are automatically scaled up or down "
+          },
+          {
+            "title": "Create IBM Code Engine Custom Domain Mappings",
+            "description": "Creates mappings from your own custom domain to your code engine application within a project."
+          },
+          {
+            "title": "Create IBM Code Engine Secrets",
+            "description": "Provides a method to include sensitive configuration information, such as passwords or SSH keys, to your deployment."
+          },
+          {
+            "title": "Create IBM Code Engine Config Maps",
+            "description": "Provides a method to include non-sensitive data information to your deployment."
+          },
+          {
+            "title": "Create IBM Code Engine Service Bindings",
+            "description": "Creates service bindings to provide application's access to IBM Cloud services."
+          }
+        ],
+        "flavors": [
+          {
+            "label": "apps",
+            "name": "apps",
+            "install_type": "fullstack",
+            "working_directory": "solutions/apps",
+            "iam_permissions": [
+                {
+                    "role_crns": [
+                      "crn:v1:bluemix:public:iam::::serviceRole:Writer"
+                    ],
+                    "service_name": "codeengine"
+                },
+                {
+                    "role_crns": [
+                      "crn:v1:bluemix:public:iam::::role:Editor"
+                    ],
+                    "service_name": "codeengine"
+                }
+            ],
+            "architecture": {
+              "descriptions": "This architecture creates IBM Code Engine entities.",
```

```suggestion
              "descriptions": "This architecture creates IBM Cloud Code Engine entities.",
```

entities is probably not the most understandable term here. Maybe include important stuff from the long_description: `configures the project, apps, domain mappings, secrets, configmaps, and service bindings.`

"Captions can help all users better understand content, but are especially helpful to users of assistive technologies. Draw attention to important features in the diagram."
from https://github.ibm.com/GoldenEye/documentation/blob/master/module-content.md#example-level-properties

---

updated text for both